appeals/api/src/server/endpoints/appellant-cases/appellant-cases.controller.js

```diff
@@ -84,6 +102,19 @@ const updateAppellantCaseById = async (req, res) => {
 		}
 	}
 
+	try {
```

Comment:
This block should not be here, otherwise the same email will be sent for all requests...
I suggest to add the following in `appellant-cases.service.js`, line 68

`if (isOutcomeValid(validationOutcome.name)) {
			try {
				await req.notifyClient.sendEmail(config.govNotify.template.appealConfirmed, recipientEmail, {
					recipient_name: recipientName,
					appeal_reference_number: appeal.reference,
					site_address: siteAddress
				});
			} catch (error) {
				if (error) {
					logger.error(error);
					return res.status(500).send({ errors: { body: ERROR_FAILED_TO_SEND_NOTIFICATION_EMAIL } });
				}
			}
		}`